chainer/datasets/tuple_dataset.py

```diff
@@ -0,0 +1,37 @@
+import six
+
+
+class TupleDataset(object):
+
+    """Dataset of a tuple of datasets.
+
+    It combines multiple datasets into one dataset. Each example is represented
+    by a tuple whose i-th item corresponds to the i-th dataset.
```

Comment:
What type of quotes should I use? I think this i is just a part of the sentence, not a specific symbol in the API... Do I have to use math quotes?